gcc/rust/checks/errors/borrowck/ffi-polonius/src/lib.rs

```diff
@@ -16,13 +16,29 @@
 // along with GCC; see the file COPYING3.  If not see
 // <http://www.gnu.org/licenses/>.
 
+#![feature(extern_types)]
```

Comment:
```suggestion
#![feature(extern_types)]

```

nit, feel free to ignore